uaclient/defaults.py

```diff
@@ -43,3 +43,9 @@
     "ua_log_level",
     "ua_security_url",
 ]
+ua_scoped_proxy_options = ("ua_apt_http_proxy", "ua_apt_https_proxy")
+global_scoped_proxy_options = (
+    "global_apt_http_proxy",
+    "global_apt_https_proxy",
+)
+deprecated_global_scoped_proxy_options = ("apt_http_proxy", "apt_https_proxy")
```

Comment:
I don't think we need these because they were added to UAConfig

Reply:
Yeah. On it.